scripts/release.sh

```diff
@@ -103,19 +106,17 @@ function release_images() {
 }
 
 function tag_images() {
-    local images="$@"
-
     # Creating a local tag so that images are uploaded with it
     git tag -a -f "${release['version']}" -m "${release['version']}"
 
-    release_images "$images --tag ${release['version']}"
+    release_images "$* --tag ${release['version']}"
```

Comment:
This one is a bit confusing, but for other reviewers, `release_images` only considers one argument, so everything has to go inside a single set of double quotes and using `$*` is fine.